docs/2.7.0/docs/daml-finance/reference/patterns.rst

```diff
@@ -6,6 +6,44 @@ Patterns
 
 This page explains some common design patterns used in the Daml Finance library.
 
+.. _factory-concept:
+
+Factory concept
+---------------
+
+Factories are helper contracts that are used to create instruments, holdings and other important
+contracts like lifecycle rules and effects. The reason why using factories is a recommended pattern
```

Comment:
We don't have a factory for lifecycle rules or effects as far as I can remember 😓